src/Queue/MongoQueue.php

```diff
@@ -1,11 +1,14 @@
 <?php
 
+declare(strict_types=1);
+
 namespace MongoDB\Laravel\Queue;
 
 use Carbon\Carbon;
 use Illuminate\Queue\DatabaseQueue;
 use MongoDB\Laravel\Connection;
 use MongoDB\Operation\FindOneAndUpdate;
+use StdClass;
```

Comment:
```suggestion
use stdClass;
```

Will also affect any usages.